gate/gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/Front50Service.java

```diff
@@ -47,8 +47,8 @@ Call<List<Map>> getApplicationHistory(
   @GET("pipelines")
   Call<List<Map>> getAllPipelineConfigs();
 
-  @GET("pipelines/{app}")
-  Call<List<Map>> getPipelineConfigsForApplication(
+  @GET("/pipelines/{app}")
```

Comment:
This shouldn't impact but this just bit me other day. "/" vs "" as had missed the prefix in a pR and it lead to a conflict.  :) JUST a "callout" - what I tested, it had no impact.    Absolutely like the more specific reference - also fixed a weird object mapper bug I was hitting!

Reply:
Good catch...https://github.com/spinnaker/spinnaker/pull/7159 removed all the leading slashes, so lemme leave it that way.

Reply:
done in https://github.com/spinnaker/spinnaker/pull/7314/commits/18c32d5f69c687c798f607ee84bcbefde202b1a9